db/migrate/20150604000000_add_permissions_to_authors.rb

```diff
@@ -0,0 +1,7 @@
+class AddPermissionsToAuthors < ActiveRecord::Migration
+  def change
+    add_column "#{Landable.configuration.database_schema_prefix}landable.authors", :read_access, :boolean, null: :false
+    add_column "#{Landable.configuration.database_schema_prefix}landable.authors", :edit_access, :boolean, null: :false
+    add_column "#{Landable.configuration.database_schema_prefix}landable.authors", :publish_access, :boolean, null: :false
+  end
+end
```

Comment:
I'm just concerned because we'll effectively be creating a second source of truth for permissions. (The first source is LDAP)